bin/queue

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Application;
+use Yiisoft\Di\Container;
+use Yiisoft\Di\ContainerConfig;
+
+require_once './vendor/autoload.php';
+
+/** @var array $definitions */
+$definitions = require_once 'definitions.php';
+
+$containerConfig = ContainerConfig::create()->withDefinitions($definitions);
+$container = new Container($containerConfig);
+
+if ($container->has(LoggerInterface::class) === false) {
+    throw new RuntimeException(
```

Comment:
Directly usage `./vendor/bin/queue` always throw this exception.

Reply:
Yes, you must configure the queue, and the logger for it to work.

https://github.com/yiisoft/yii-queue/tree/use-yii-console-optional#usage-with-symfony-console

Reply:
Use nulllogger. Otherwise utility is no matter.

Reply:
Done.

Reply:
Now check on `LoggerInterface` not need.

Reply:
And explain to me how I am going to check that LoggerInterface is not necessary, if the queue uses it?